packages/typespec-azure-resource-manager/lib/operations.tsp

```diff
@@ -694,3 +694,113 @@ op checkLocalNameAvailability<
   Response,
   AdditionalParams
 >;
+
+/**
+ * @dev An action performed at subscription level.
+ * @template Request The request model for the action.
+ * @template Response The response type for the action.
+ * @template Parameters Optional. Additional parameters after the path parameters (e.g. Location)
+ * @template Scope Optional. The scope of the action.
+ * @template Error Optional. The error response, if non-standard.
+ */
+@post
+@action
+@armResourceCollectionAction
+op ArmProviderActionSync<
+  Request extends {} | void = void,
+  Response extends {} | void = void,
+  Parameters extends {} = {},
+  Scope extends {} = SubscriptionActionScope,
```

Comment:
```suggestion
  Scope extends SubscriptionActionScope = SubscriptionActionScope,
```

I think we need to nail this down, it is only going to work for tenant and subscription actions, and we can always open this up later if we need it

Reply:
I did all this changes on my latest commit, I will proceed to resolve these comments, thanks Mark